IsoLib/libisomediafile/src/MP4MovieFile.c

```diff
@@ -378,6 +381,12 @@ ISOStartMovieFragment(MP4Movie theMovie)
     moof              = (MP4MovieFragmentAtomPtr)movie->moovAtomPtr;
     mfhd              = (MP4MovieFragmentHeaderAtomPtr)moof->mfhd;
     fragment_sequence = mfhd->sequence_number + 1;
+    for(i = 0; i < moof->atomList->entryCount; i++)
+    {
+      err = MP4GetListEntry(moof->atomList, i, (char **)&traf);
+      if(err) goto bail;
+      traf->destroy((MP4AtomPtr)traf);
+    }
```

Comment:
Not so sure about this one. Below moof->destroy is called which should already do this.

Reply:
I understand this library's design and understand what you said. 
But, If there is no below code
```
for(i = 0; i < moof->atomList->entryCount; i++)
    {
      err = MP4GetListEntry(moof->atomList, i, (char **)&traf);
      if(err) goto bail;
      traf->destroy((MP4AtomPtr)traf);
    }
```
traf is still in memory.

If this code is not matching this library design, 
you need to check moof destory's flow.